account_invoice_hour_overview/account_invoice.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+# © 2017 Sunflower IT (http://sunflowerweb.nl)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from itertools import groupby
+import time
+from datetime import datetime as dt
+
+from openerp import fields, models, api, _
+import openerp.addons.decimal_precision as dp
+
+
+class AccountInvoice(models.Model):
+    _inherit = 'account.invoice'
+
+    @api.multi
+    def _get_report_analytic_lines(self):
+        for invoice in self:
+            lines = self.env['hr.analytic.timesheet'].search([
+                ('invoice_id', '=', invoice.id)
+            ])
+            invoice.report_analytic_lines = lines
+
+    report_analytic_lines = fields.One2many('hr.analytic.timesheet',
```

Comment:
why to you need to compute this and don't just use a one2many linked by `invoice_id`?